packages/jest/src/lib/error.ts

```diff
@@ -20,6 +20,7 @@ import path from 'path';
 import StackUtils from 'stack-utils';
 import url from 'url';
 import { ErrorSchema, LocationSchema } from '../types';
+import debug from 'debug';
```

Comment:
```suggestion
import { debug } from '../lib/debug';
```

This adds our custom prefix, so it shows up with the debug instructions we give in the readme.